### PR TITLE
UI/mobile responsive

### LINKED ISSUE #55: UI not responsive for smaller Screen sizes
## Describe the bug

UI is not responsive for smaller screen sizes 

## To Reproduce

1. Open application chrome
2. Use chrome inspect element to checkout responsiveness
3. Switch between screens sizes. 

## Environment (please complete the following information):

- OS: Windows 10
- Browser: Chrome

**Additional context (OPTIONAL)**

Screenshot 1          |  Screenshot 2             |  Screenshot 3            |  Screenshot 4     
:-------------------------:|:-------------------------:|:-------------------------:|:-------------------------:|
ui 1|ui 2|ui3|ui 4

### PR DESCRIPTION
**Pull request by AsavariA**
**Issue Number:  #55**
**Issue name: UI not responsive for smaller Screen sizes**

fixes #55 

**Made pages of the app mobile responsive**

Enhanced app UI such that it looks good on small size devices too.

**Checklist**

- [x] My code follows the code style of this project.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: src/assets/scss/components/storiesList.scss [hunk 1]
```diff
@@ -48,11 +48,80 @@
...
+    margin: 0px;
+    width: 100%;
+  }
+
+  .story-small-details {
+    width: 100%;
+  }
+}
+
+// @media only screen and (max-width: 768px) {
```
Comment (cyntss) [inline src/assets/scss/components/storiesList.scss:93]: let's remove these commented lines

File: src/assets/scss/components/timeline.scss [hunk 2]
```diff
@@ -1,11 +1,12 @@
 .status-wrapper {
-  max-height: 80px;
+  // max-height: 80px;
```
Comment (cyntss) [inline src/assets/scss/components/timeline.scss:2]: let's remove these commented lines

File: src/assets/scss/pages/home.scss [hunk 3]
```diff
@@ -18,16 +18,37 @@ $subheader-font-size: 18px;
...
   align-items: center;
   background-color: $eos-white;
   border-radius: 10px;
-  padding: 8px 20px;
+  padding: 8px;
 }
 
-.link-default {
-  padding-left: 0.5em;
+// .link-default {
```
Comment (cyntss) [inline src/assets/scss/pages/home.scss:36]: let's remove these commented lines

**Review by cyntss — CHANGES_REQUESTED** [review]

**Comment by cyntss** [comment]
I tested the UI and it works beautifully! excellent PR @AsavariA 
Once you cleanup those comments in the SCSS we can merge this

**Comment by AsavariA** [comment]
> I tested the UI and it works beautifully! excellent PR @AsavariA
> Once you cleanup those comments in the SCSS we can merge this

@cyntss I have made the changes mentioned. Thank you :D

**Review by sharmaaditya570191 — APPROVED** [review]

**Comment by sharmaaditya570191** [comment]
Awesome work @AsavariA. 🎉 

Feel free to join us on Slack via link in README for interacting with other contributors.

**Pull request merged**